examples/set-namespace-depends-on/.expected/diff.patch

```diff
@@ -34,3 +35,8 @@ index e45cae5..2ff45d5 100644
  ---
  apiVersion: rbac.authorization.k8s.io/v1
  kind: ClusterRoleBinding
+@@ -43,3 +43,4 @@ subjects:
+ - apiGroup: rbac.authorization.k8s.io
+   kind: Group
+   name: admin
++  namespace: example-ns
```

Comment:
Can you please confirm if this change is expected?

Reply:
you are absolutely right. updated.